server/konnectors/edf.coffee

```diff
@@ -1143,20 +1204,42 @@ displayData = (requiredFields, entries, data, next) ->
 
     next()
 
+fetchEdeliaData = (requiredFields, entries, data, next) ->
```

Comment:
OK. Are you sure there is no interference using the fetcher inside the fectcher like notifications beging thrown twice? If not, let's go this way.